docs/administration/configuration/configuring-compute-backends/index.md

```diff
@@ -0,0 +1,150 @@
+  If a user job exceeds this limit, the user job will be terminated and an
+  error reported back to the user. By default the runtime limit is about 7
+  days. Please set this variable based on the typical user workflow runtime
+  needs in your installation.
+
+- The maximum runtime limit a user can request is governed by the Helm value
+  [`kubernetes_jobs_max_user_timeout_limit`](https://github.com/reanahub/reana/tree/master/helm/reana).
+  For example, you can allow users to request a runtime of up to 14 days even
+  though the default is 7 days. If a user would request more than this value,
+  the job would not be accepted.
+
+Here is a customisation snippet example:
+
+```yaml
+compute_backends:
+  - kubernetes
+kubernetes_jobs_memory_limit: 2Gi
+kubernetes_jobs_max_user_memory_limit: 10Gi
+```
+
+## Slurm
+
+The Slurm compute backend is useful for high-performance user computing needs.
+Currently REANA supports the Slurm batch system at CERN (`slurmcern`).
+
+If you wish to include the Slurm compute backend in your installation, please
+include the `slurmcern` value in the Helm list
+[`compute_backends`](https://github.com/reanahub/reana/tree/master/helm/reana)
+and customise the
+[`components.reana-job-controller.image`](https://github.com/reanahub/reana/tree/master/helm/reana)
+value to use an image compiled with the Slurm CERN backend support, such as
+`reanahub/reana-job-controller-htcondorcern-slurmcern:0.9.0`.
+
+If you would like to further customise the Slurm compute backend, you can set
+various environment variables for the `reana-job-controller` component that
+will affect the scheduling of user job pods:
+
+- `SLURM_HOSTNAME` is the FQDN of the Slurm cluster head node. It is used to
+  submit user jobs. (Default: `hpc-batch.cern.ch`.)
+
+- `SLURM_HEADNODE_PORT` is the port on the head node where to connect.
+  (Default: 22.)
+
+- `SLURM_PARTITION` is the name of the default Slurm queue used for user jobs.
+  (Default: `inf-short`.) Note that users can select any other partition they
+  have access to using the workflow hints described in the user-oriented
+  documentation linked above.
+
+- `SLURM_JOB_TIMELIMET` is the default runtime limit for Slurm jobs. (Default:
+  60.) Note that users can override this value by workflow hints described in
+  the user-oriented documentation linked above.
+
+- Finally, you can set various internal TCP connection timeouts affecting the
+  communication between REANA and the Slurm head node. It is usually not
+  necessary to alter these. `SLURM_SSH_TIMEOUT`, `SLURM_SSH_BANNER_TIMEOUT`,
+  and `SLURM_SSH_AUTH_TIMEOUT`. Default: 60 seconds each.
+
+Here is a customisation snippet example:
+
+```yaml
+compute_backends:
+  - kubernetes
+  - slurmcern
+components:
+  reana_job_controller:
+    image: reanahub/reana-job-controller-htcondorcern-slurmcern:0.9.0
+    environment:
+      SLURM_PARTITION: "batch-short"
+      SLURM_JOB_TIMELIMIT: 120
```

Comment:
I think this would not work, as reana-job-controller is spawned by reana-workflow-controller, which does not pass `SLURM_*` variables along ([source code](https://github.com/reanahub/reana-workflow-controller/blob/959be5cb9ca6eb90d9e063f754a2d8b97f2fad50/reana_workflow_controller/workflow_run_manager.py#L543-L596)). As far as I can tell, there is no way to customise these at the moment. Should I open an issue about this?

Reply:
Good catch :+1: Considering that we may also want to edit `r-commons` and friends to introduce the new Slurm-related environment variables, and that we don't know whether this would be a sufficient change for non-CERN deployments, I've left this part out for the time being.